docs/docsite/rst/roadmap/COLLECTIONS_8.rst

```diff
@@ -40,6 +40,12 @@ Release schedule
 
   Breaking changes will be introduced in Ansible 8.0.0, although we encourage the use of deprecation periods that will show up in at least one Ansible release before the breaking change happens, this is not guaranteed.
 
+.. note::
+
+  When a prerelease of Ansible 8 happens in the same week as an Ansible 7 release, the Ansible 8 prerelease will happen on the day after the Ansible 7 release.
+  When a release of Ansible 8 happens in the same week as an Ansible 9 release, the Ansible 8 release will happen on the day after the Ansible 9 release.
+  In general it's in the discretion of the release manager to delay a release by 1-2 days for reasons such as personal (schedule) problems, technical problems (CI/infrastructure breakdown), and so on.
```

Comment:
Usually Ansible releases are always scheduled on the same day, since they happen one day after a corresponding ansible-core release.

Let me create a suggestion further below.

Reply:
> Usually Ansible releases are always scheduled on the same day, since they happen one day after a corresponding ansible-core release.

Yes, _usually_. But they might also delay a release for personal (schedule) or technical problems.